token/program-2022/src/extension/permanent_delegate.rs

```diff
@@ -0,0 +1,18 @@
+use {
+    crate::{
+        extension::{Extension, ExtensionType},
+        pod::*,
+    },
+    bytemuck::{Pod, Zeroable},
+};
+
+/// Permanent delegate extension data for mints.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Pod, Zeroable)]
+pub struct PermanentDelegate {
+    /// Optional authority to close the mint
+    pub delegate: OptionalNonZeroPubkey,
```

Comment:
this comment needs to be updated

Reply:
Man, I really thought I caught all of those.  thanks!